pages/portfolio.html

```diff
@@ -5,8 +5,71 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link rel="stylesheet" href="/stylesheets/style.css">
 </head>
 <body>
-  
+  <header>
+    <h1>my portfolio</h1>
+    <nav>
+        <ul>
+            <li><a href="index.html">home</a></li>
+            <li><a href="about.html">about</a></li>
+        </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="portfolio">
+      <h2>Personal Portfolio Project</h2>
+      <a href="https://github.com/eggoweggo/personal-portfolio-site">
```

Comment:
you could add the target attribute to your links and it will open them in a new tab. This keeps the user on your page and be able to access other content 

an example is here https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_a_target